package.json

```diff
@@ -25,16 +25,32 @@
   "author": "Brainhub",
   "license": "MIT",
   "dependencies": {
+    "@types/bcrypt": "1.0.0",
     "@types/bunyan": "1.8.4",
+    "bcrypt": "2.0.0",
```

Comment:
Such dependencies should be defined in package.json of specific package. Same with `jsonwebtoken`